docs/guides/device-gateway/arm64.md

```diff
@@ -0,0 +1,163 @@
+
+1. Start the TCP tunnel with `ngrok`.
+
+   ::::warning
+   TCP endpoints are only available on a free plan after [adding a valid payment method](https://dashboard.ngrok.com/settings#id-verification) to your account. If you get a different error, ensure your authtoken is configured correctly.
+   ::::
+
+   ```bash
+   ngrok tcp 22
+   ```
+
+   When the ngrok agent starts, you'll see a `Forwarding` line with similar information about the public endpoint for the TCP tunnel to your device:
+
+   ```bash
+   Forwarding                    tcp://4.tcp.us-cal-1.ngrok.io:16420 -> localhost:22
+   ```
+
+1. On your local workstation, SSH into your ARM64 device with the command below, replacing the variables with the following:
+
+   - `{NGROK_PORT}`: The port number assigned to your ngrok agent (e.g. if the agent shows `tcp://4.tcp.us-cal-1.ngrok.io:12345`, your port number is `12345`).
+   - `{USER}`: An existing user on your remote device.
+   - `{NGROK_TCP_ADDRESS}` | The address of the ngrok agent (e.g. if the agent shows `tcp://4.tcp.us-cal-1.ngrok.io:12345`, your TCP address is `4.tcp.us-cal-1.ngrok.io`).
+     <br />
+
+   ```bash
+   ssh -p {NGROK_PORT} {USER}@{NGROK_TCP_ADDRESS}
+   ```
+
+## Step 3: Enable ingress to a service on your ARM64-based Linux device
+
+Separately from SSH access, you can also use ngrok to create an [HTTP tunnel](/docs/http/index.mdx) to route traffic to specific applications or services running on your ARM64 device's `localhost` network.
+
+:::note
+If you already established a TCP tunnel for SSH access, you'll either need to create a second tunnel for the HTTP tunnel, or use the [agent configuration file](/docs/agent/config.mdx) to define multiple tunnels.
+:::
+
+1. Use ngrok to create an HTTP tunnel at the port on which your service operates, e.g. port `8080`.
+
+   ```bash
+   ngrok http 8080
+   ```
+
+   When the ngrok agent starts, you'll see a `Forwarding` line with similar information about the public endpoint for the HTTP tunnel to your device:
+
+   ```bash
+   Forwarding                    https://12345.ngrok.app -> http://localhost:8080
+   ```
+
+1. Optionally, you can reserve a [static subdomain](/docs/network-edge/domains-and-tcp-addresses.mdx) on `ngrok.app` or `ngrok.dev` like so:
+
+   ```
+   ngrok http 8080 --domain example.ngrok.app
+   ```
+
+1. Open a browser and navigate to the endpoint, e.g. `https://12345.ngrok.app`, to access the service from any remote system.
+
+## Step 4: Add a Traffic Policy to restrict IPs
+
+Now that you have SSH tunneling and service ingress handled via ngrok, you may also want to protect those services from unknown and untrusted users. You'll use the Traffic Policy module, which works with both [TCP](/docs/tcp/traffic-policy/index.mdx) and [HTTP](/docs/http/traffic-policy/index.mdx) tunnels, along with the [Restrict IPs](/docs/http/traffic-policy/actions/restrict-ips.mdx) action.
```

Comment:
Update links [TCP](/docs/tcp) and [HTTP](docs/http)

Reply:
Hey! I restructured this a bit to make it clearer that I wanted folks to be able to see the specific Restrict IP action docs for both HTTP and TLS tunnels. I believe the functionality is a little different, so I didn't want to show only the HTTP version.